src/stories/inputs/DynamicField/ControlsPreview.tsx

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) TotalSoft.
+// This source code is licensed under the MIT license.
+
+import React, { useState } from 'react'
+import { Grid, Switch } from '@mui/material'
+import { Autocomplete, DynamicField, Typography } from 'components'
+import { ControlType } from 'components/inputs/DynamicField/types'
+import { SwitchProps } from '@mui/material'
+import { controlTypes } from './_mocks'
+import { options } from '../Autocomplete/_mocks'
+import { useDynamicProps } from './_hooks'
+import { controlsStyles } from './_styles'
+import { makeStyles } from '@mui/styles'
+import { InfoLink } from './ControlPreview'
+
+const useStyles = makeStyles(controlsStyles)
+
+const DefaultPreview = () => {
+  const classes = useStyles()
+
+  const [value, setValue] = useState<unknown>()
+  const [type, setType] = useState<ControlType>(ControlType.Text)
+
+  const { controlValues, props, currentControlData, handlePropsChange } = useDynamicProps(type)
+
+  return (
+    <Grid container spacing={2} className={classes.controlContainer}>
+      <Grid item container justifyContent={'center'} alignItems={'center'} sx={{ gap: '.5rem', marginBottom: '1rem' }}>
+        <Typography fontWeight={700}>Control type:</Typography>
+        <Autocomplete
+          options={controlTypes}
+          simpleValue
+          value={type}
+          onChange={value => {
+            setValue(null)
+            setType(value)
+          }}
+        />
+      </Grid>
+      <Grid item xs={12} container alignItems={'center'} spacing={5}>
+        <Grid item container justifyContent={'center'} xs={12} sm={currentControlData.props.length > 0 ? 6 : 12}>
```

Comment:
Poate reusesti sa mai scapi din Grid-uri.

Reply:
Cred ca e minimul necesar deja. Fiecare item are nevoie de un container. Mie oricum mi se par cam naspa Grid-urile lor, mai ales cu spacing-ul, foarte urat de lucrat cu ele. As fi mers cu div-uri si stiluri, dar aia e.

Reply:
Am facu un push pe ControlsPreview cu sugestia mea de simplificare pt Grid-urile de mai sus. Poti arunca un ochi si daca ti se pare ok ar trebui facut si pt ControlPreview.

Reply:
![image](https://github.com/osstotalsoft/rocket-ui-ts/assets/35111313/c6d0fd76-bfdc-487b-81d9-f0da348965a2)
E o problema cu responsiveness. Am fixat eu asta. In regula, poate ramane asa